ecs_deploy/cli.py

```diff
@@ -37,6 +37,7 @@ def get_client(access_key_id, secret_access_key, region, profile):
 @click.option('-x', '--execution-role', type=str, help='Sets the execution\'s role ARN: <execution role ARN>')
 @click.option('--task', type=str, help='Task definition to be deployed. Can be a task ARN or a task family with optional revision')
 @click.option('--region', required=False, help='AWS region (e.g. eu-central-1)')
+@click.option('--env-file', type=(str, str), default=((None, None),), multiple=True, required=False, help='load environment variables from file')
```

Comment:
Please move this up to the `-e` option, then we have this logically group in the same context

Reply:
please make it a bit more specific, what kind of file it can read - maybe just ".env file" instead of "file"

Reply:
Thanks for the feedback!

Regarding testing of the helper function and set_environment: since these functions read and use parameters from a file, I was thinking that the best way to test this might be to create a temporary .env file using the [tempfile](https://docs.python.org/3/library/tempfile.html) library, but it turns out that this might not be possible in the way I envisioned. I might find a workaround if I keep debugging though.

An alternative way (which might save me some hours of headache) is to create a simple .env-file under your tests folder, but I'm not sure if you want that in your repo. It's up to you!

Reply:
I think I would go for a temporary file, too. The .env file format is pretty straight forward - there are only four rules (https://docs.docker.com/compose/env-file/) - which you also covered in your helper, right? Do you need a template library or can we just go with some pre-defined valid and invalid file contents?